Samples/All/BumbleRumble/Common/NetworkManager.cpp

```diff
@@ -1224,3 +1199,74 @@ PartyChatControl* NetworkManager::GetChatControl(std::string& peer)
     }
     return nullptr;
 }
+
+void NetworkManager::SetCognitiveServicesEnabled(bool bEnabled)
+{
+	m_enableCognitiveServices = bEnabled;
+
+	// For the purposes of the sample, enable or disable all possible features
+	SetTextChatTranslationOptions(bEnabled);
+	SetVoiceChatTranscriptionOptions(bEnabled, bEnabled, bEnabled, false, bEnabled); 

```

Comment:
Per conversation, I think it's important that speech-to-text transcription, speech-to-text translation, text-to-speech, and text-to-text translation all be tied to different UI toggles. The previous behavior was also "just" for sample purposes, but it was still copied verbatim anyways.

(It's okay for TranscribeSelf/TranscribeOtherChatControlsWithMatchingLanguages/TranscribeOtherChatControlsWithNonMatchingLanguages to all be tied to a single speech-to-text transcription option.)

Reply:
I do not like this pattern of having tons of booleans.  better might be to have an enum list and OR them together as one param.